presto-main/src/main/java/com/facebook/presto/util/MoreMath.java

```diff
@@ -110,4 +114,14 @@ public static double firstNonNaN(double... values)
         }
         throw new IllegalArgumentException("All values are NaN");
     }
+
+    public static double roundToDouble(double value)
```

Comment:
add a unit test for this new helper function

Reply:
There is one, in TestMoreMath